src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/ProtoTest.kt

```diff
@@ -0,0 +1,13 @@
+package com.bardsoftware.papeeria.backend.cosmas

```

Comment:
Please add the same copyright notice as in `cosmas.proto` in the very beginning of this file

Reply:
done